lib/internal/test_runner/runner.js

```diff
@@ -733,7 +735,6 @@ function run(options = kEmptyObject) {
     };
   } else if (isolation === 'none') {
     if (watch) {
-      const absoluteTestFiles = ArrayPrototypeMap(testFiles, (file) => (isAbsolute(file) ? file : resolve(cwd, file)));
```

Comment:
Instead of resolving absolute paths 2 times, one for the coverage and other for this specific watch, I resolved them 1 time. Still needed to delete the `isAbsolute(file)` part to clean weird absolute paths